### PR TITLE
[SIS-047] Convert svgs to react native svg transformer

### LINKED ISSUE #47: [SIS-047] Convert SVGs to react-native-svg-transformer
- [x] Install [react-native-svg-transformer](https://www.figma.com/file/kSZduD0A5hk8sHmvqiC7DE/Sistema-Hi-Fidelity-Designs-%2B-Design-System?node-id=180%3A15386&t=uBWLplRiS9RKkpMh-0) and make sure build is working
- [ ] Convert all SVGs to use the package
- [x] Add a section in the Wiki on how to use it. Can duplicate [last year's Period Purse](https://github.com/uoftblueprint/the-period-purse/wiki/SVG-Color-Change) wiki page, go thank Kenneth. 
- [ ] Let your fellow developers know about the changes!

### PR DESCRIPTION
**Pull request by kurtislaw**
# Description

I installed the `react-native-svg-transfomer` package and changed the imports in the files under `src/editor/components` to work with SVG files instead.

Ticket #47 

## Type of change

Please delete options that are not relevant.

- [ ] Documentation changes 
- [ ] Build (changing build dependencies, updating packages, etc.)

# Checklist:

- [x] My PR title is formatted as [SIS-\<issue number\>] \<issue name\> (eg. [SIS-010] Create UI button)
- [x] I have linked my issue/ticket on the project board to this PR
- [x] I have sent my PR to #team-sistema-prs on Slack and requested two people to review
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings
- [x] I have tested that my feature works as expected on Android and iOS. If you are a Windows dev, please request a Mac dev to run your branch. If it's a front-end change, I have attached screenshots of what the result looks like on both platforms.

### CODE REVIEW

**Comment by ramyzhang** [comment]
i'll review this after it's been reviewed by @Lemiex @BakerWJ !!! :D

File: src/editor/components/LessonPlanNotes.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Hidden } from '@mui/material';
+// import { Hidden } from '@mui/material';
```
Comment (yuemily24) [inline src/editor/components/LessonPlanNotes.js]: Not sure what this is, can it be safely deleted since it's commented out?

**Review by yuemily24 — APPROVED** [review]
Nice, thanks so much for doing this ticket! Works on Android, going to assume it works on iOS as well since I see all the pod changes lol

File: src/editor/components/LessonPlanNotes.js (hunk 1 above)
Comment (yuemily24) [reply src/editor/components/LessonPlanNotes.js]: Got rid of it in a commit.

**Pull request merged**